js/base/Exchange.js

```diff
@@ -1743,6 +1743,27 @@ module.exports = class Exchange {
         return networkCode;
     }
 
+    networkCodesToIds (networkCodes = undefined) {
```

Comment:
@samgermain First the PR title/description does not match its content (adding networkCodesToIds)

Second,`networkCodeToIDs` should only call `networkToId` inside a loop to parse the N number of networkCode. We should do that because `networkToId` has additional logic ( and it will probably get more in the future), and we want to avoid having to duplicate it here

Reply:
@samgermain up 😅